clang and the spirv-llvm translator install to /usr/bin, so add it to the path
here
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9773b2c..ec0ff96 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,6 +83,8 @@ execute_process( COMMAND ${LLVM_CONFIG} "--bindir"
       OUTPUT_VARIABLE LLVM_BINDIR
       OUTPUT_STRIP_TRAILING_WHITESPACE )
 
+list( APPEND LLVM_BINDIR "/usr/bin" )
+
 # These were not properly reported in early LLVM and we don't need them
 list( APPEND LLVM_CXX_FLAGS -fno-rtti -fno-exceptions )
 
